### ISSUE TITLE
No results and no error in query select with 21 columns or more

### ISSUE THREAD

**Issue by ajoses** [nakagami/firebirdsql#65]
Hi,

Whe you have a query with two tables and in the select you put specific columns of both tables, the Query function return no result, if I use and select * from both tables then it works.

I attach a program and a example database where I reproduce this error.

All the querys of the file where tested with flamerobin and run correctly, however I don't know why in the go program doesn't work.

Any comment to where I can look for the error will be appreciated.

[PRUEBAS.zip](https://github.com/nakagami/firebirdsql/files/2214987/PRUEBAS.zip)
[main.zip](https://github.com/nakagami/firebirdsql/files/2214990/main.zip)

**Comment by ajoses** [comment]
My version of go:
go version go1.10.3 linux/amd64

Firebird: 
LI-V2.5.7.27050 Firebird 2.5

**Comment by ajoses** [comment]
Hi,
I ran some more tests, and found that the error is the number of columns selected, if there are more than 21 then it not return the value even thought the query has results.

I attached another serie of tests where I tested with one relation and with two relation.

I tried to find the code with the problem, but with no success.

Any orientation will be appreciated.
[main.zip](https://github.com/nakagami/firebirdsql/files/2215276/main.zip)
